### PR TITLE
Comma, middle click, cyrillian

### PR DESCRIPTION
**Pull request by stefandanzl**
Better fix for #972

### CODE REVIEW

**Comment by stefandanzl** [comment]
This should now also allow for cyrillian characters for DB property ids #1024 

Files can opened in new pane on middle click #973

**Comment by RafaelGB** [comment]
hello!
I have seen that you have dedicated a lot of effort both in reading issues and in the PR. Thank you so much!

There is a lot of work accumulated in this repository since Obsidian included the databases in its roadmap. I have not wanted to continue the project until I know how the native databases and their views will work.

While I continue learning and developing another plugin ([Zettelflow](https://github.com/RafaelGB/Obsidian-ZettelFlow)) in which I have gotten good ideas and little by little I will be able to adapt the dbfolder project with another approach

File: src/services/MarkdownRenderService.ts [hunk 1]
```diff
@@ -37,15 +37,34 @@ class MarkdownRenderService {
...
             // Array modifiers
-            if (array && SUGGESTER_REGEX.TEXT_ARRAY.test(markdownString)) {
-                let alternativeString = "";
-                markdownString
-                    .replaceAll(SUGGESTER_REGEX.TEXT_ARRAY, "$2$3$4")
-                    .split(",")
-                    .forEach((item) => {
-                        alternativeString = alternativeString.concat(`- ${item.trim()}\n`);
+            if (array && SUGGESTER_REGEX.TEXT_ARRAY.test(markdownString) && !SUGGESTER_REGEX.IS_MD_LINK.test(markdownString) && !SUGGESTER_REGEX.IS_URL_LINK.test(markdownString)) {
+                // let alternativeString = "";
```
Comment (RafaelGB) [inline src/services/MarkdownRenderService.ts:41]: Just remove the commented code to do not accumulate code smells

File: styles.css [hunk 2]
```diff
@@ -940,6 +940,11 @@ div.database-plugin__td.data-input span:focus {
 *               React-datepicker styles
 ************************************************************/
 /** VARIABLES */
+
+.react-datepicker__aria-live {
+  display: none !important;
```
Comment (RafaelGB) [inline styles.css:945]: !important should be avoided as a good practice . If you thinks there is no alternative I will aprove the PR

File: src/components/modals/columnSettings/handlers/ColumnIdInputHandler.ts [hunk 3]
```diff
@@ -74,7 +74,8 @@ export class ColumnIdInputHandler extends AbstractHandlerClass<ColumnSettingsHan
             return t("column_settings_modal_column_id_error_empty_root_key");
         }
         // Validate special characters in root key
-        if (rootKey.match(/[^a-zA-Z0-9_]/)) {
+        // if (rootKey.match(/[^a-zA-Z0-9_]/)) {
```
Comment (RafaelGB) [inline src/components/modals/columnSettings/handlers/ColumnIdInputHandler.ts:77]: remove the old regular expression

File: src/services/MarkdownRenderService.ts (hunk 1 above)
Comment (stefandanzl) [reply src/services/MarkdownRenderService.ts]: i removed it - i hope you can see the updated version

File: styles.css (hunk 2 above)
Comment (stefandanzl) [reply styles.css:945]: sadly it's not working, is it possible that this doesn't get bundled when compiling or something like that?

Comment (RafaelGB) [reply styles.css:945]: the css its a mess. I am starting to learn sass with the other plugin to be more scalable and maintainable.

With the next minor version I will just add the skeleton and required dependencies to use it. We can go here with !importat tag then

**Review by RafaelGB — APPROVED** [review]

**Comment by RafaelGB** [comment]
In theory as contributor you can merge the PR now =) thanks for your effort !

**Comment by stefandanzl** [comment]
i used rebase commit, is that required or would that be done automatically anyways?

**Comment by RafaelGB** [comment]
There is no defined practice for commits, rebasing is fine, although with the proper git client without rebasing it is better so we don't lose information. It depends on your tastes

**Comment by stefandanzl** [comment]
after mergin your changes in my repository (changes like `npm run release` )  my plugin builds are faulty:
showing 0 for all row numbers or not rendering the table at all with following error messages:
```
Uncaught (in promise) Error: No available storage method found.
    at driverPromiseLoop
```

```
Uncaught (in promise) DOMException: Internal error opening backing store for indexedDB.open.
```

```
Uncaught (in promise) Error: No available storage method found.
    at driverPromiseLoop
```

**Comment by stefandanzl** [comment]
I did `git clone` your repository and the error message doesn't appear anymore but the row number is still always 0
image

**Pull request merged**